lib/transformer.js

```diff
@@ -36,13 +36,27 @@ module.exports = {
   },
 
   /**
+   * Create Sequelize object based on config options
+   * @param  {Object} config Trails.js store
+   * @return {Sequelize}     Sequelize instance
+   */
+  createFromConfig (config) {
+    if (config.uri) {
```

Comment:
uristore is name of store
uri is param in store config, like username, password, host, etc.
```
module.exports = {

  /**
   * Define the database stores. A store is typically a single database.
   *
   * Use the SQLite3 by default for development purposes.
   *
   * Set production connection info in config/env/production.js
   */
  stores: {
    postgresstore: {
      uri: 'postgresql://user:password@127.0.0.1/db',
      // dialect: 'postgres',
      // host: '127.0.0.1',
      // username: 'user',
      // password: 'pass',
      // database: 'db'
    }
  },
}

```